alica_engine/include/engine/blackboard/Blackboard.h

```diff
@@ -42,6 +45,18 @@ class BlackboardImpl
     bool empty() const { return vals.empty(); }
     size_t size() const { return vals.size(); }
     std::unordered_map<std::string, std::any> vals;
+
+    static std::unordered_map<std::string /* library name */, std::unique_ptr<IAlicaWorldModel>> worldModels_;
```

Comment:
Better not to add more static things.  Instead, can we store the "world model" blackboards in the engine?  The engine also has it's own blackboard that we can replace with the world model blackboards.

Reply:
yes that's better, we can store it in the AlicaEngine object, but I think we should retain the access via the blackboard API

Reply:
Any reason to access through blackboard rather than through the RunningPlan/Behavior?

Reply:
RunningPlan is a viable alternative, but I think it is already a bloated class & seems more consistent to me to access all storage via blackboard.

Reply:
via behaviour is not possible because we need a uniform way to access world model for conditions as well

Reply:
Okay